jekyll/_cci2/updating-server.adoc

```diff
@@ -1,58 +1,207 @@
-= Updating a Server Installation
+= Upgrading a Server Installation
 :page-layout: classic-docs
 :page-liquid:
 :icons: font
 :toc: macro
 :toc-title:
 
-This section describes the Server update process.
+This section describes the process for upgrading your CircleCI Server installation from v2.17 to v2.18.
 
 toc::[]
 
-== Update Steps
+== Org Rename Script
 
-. Take a snapshot of your installation so you can rollback later if necessary (optional but recommended) - steps below
-. Check you are running Docker v17.12.1 and update if necessary
-. Update Replicated to install the new version of CircleCI Server - steps below
+WARNING: Before upgrading please read and follow the steps below if you have **ever had issues with renaming an organization within CircleCI**
+or you suspect that an **organization rename might have happened at any point**.
 
-== Snapshot for Rollback
+. SSH into your Services machine
+. REPL into `workflows-conductor` by running the following: `sudo docker exec -it workflows-conductor lein repl :connect 6005`
+. Copy/paste this script into the REPL session. It will run migration and output current progress.
```

Comment:
@GERey org rename script here :-)

Reply:
I think the script as it should be exists below so I'll resolve this :) Thanks!